pkgs/development/tools/dive/default.nix

```diff
@@ -11,7 +11,11 @@ buildGoModule rec {
     sha256 = "sha256-1pmw8pUlek5FlI1oAuvLSqDow7hw5rw86DRDZ7pFAmA=";
   };
 
-  vendorSha256 = "sha256-0gJ3dAPoilh3IWkuesy8geNsuI1T0DN64XvInc9LvlM=";
+  patches = [
+    ./update-x-sys.patch
```

Comment:
Is there a reason we can't use https://github.com/wagoodman/dive/pull/402/commits/fe9411c414418d839a8638bb9a12ccfc892b5845?

If we can't for whatever reason it would be good if you opened a PR for this patch like you did for `noti`.

Reply:
Didn't search for patches but we can use that one.